spec/models/post_mover_spec.rb

```diff
@@ -887,6 +887,27 @@ def create_topic_user(user, opts = {})
             expect(new_post.post_revisions.size).to eq(1)
           end
 
+          context "with subfolder installs" do
+            before { Discourse.stubs(:base_path).returns("/forum") }
```

Comment:
We have a helper for this in specs:

```
set_subfolder "/forum"
```

An example:

https://github.com/discourse/discourse/blob/ea0b37c1cf7fe87e440bd5e665b534ffbe74025c/spec/models/category_spec.rb#L509-L515